packages/@aws-cdk/aws-lambda/test/test.event-source-mapping.ts

```diff
@@ -0,0 +1,148 @@
+/* tslint:disable */
```

Comment:
Generally not encouraged. Why is this needed?

Reply:
Orphaned from getting the basic test up - cleaning it up now